lib/src/ui/movies_page.dart

```diff
@@ -150,17 +117,37 @@ class _MoviesPageState extends State<MoviesPage> {
           ),
           child: Column(
             children: <Widget>[
-              CardSwiper(
-                trendingMovies: _swiperTrendingMovies,
-                isTextFieldEmpty: widget.bloc.isTextFieldEmpty,
+              StreamBuilder<StateType>(
+                initialData: _getInitialData(),
+                stream: widget.bloc.cardSwiperMoviesStream,
+                builder: (context, AsyncSnapshot<StateType> snapshot) {
+                  return CardSwiperEvents(
+                    state: snapshot.data!,
```

Comment:
where do you check if snapshot has data?

Reply:
done